backend/webserver/models.py

```diff
@@ -15,3 +16,61 @@ class FollowRequest(models.Model):
 class Follow(models.Model):
     follower = models.ForeignKey(Author, related_name='following_authors', on_delete=models.CASCADE)
     followee = models.ForeignKey(Author, related_name='followed_by_authors', on_delete=models.CASCADE)
+
+class Post(models.Model):
+
+    author = models.ForeignKey(Author,on_delete=models.CASCADE)
+
+    created_at = models.DateTimeField(verbose_name="date created",null=False, blank=False, default = datetime.now)
+
+    edited_at = models.DateTimeField("date edited")
```

Comment:
```suggestion
    edited_at = models.DateTimeField("date edited", null=True)
```
- If this field is `null`, it means that the author has not edited the post